CHANGELOG.md

```diff
@@ -4,6 +4,10 @@
 
 ---
 
+## 1.1.1
```

Comment:
Can we leave this until 1.1.1 is actually released? Instead can you add the pending changes to the pending section above?

Reply:
Moved to pending section

Reply:
Can you remove the #1.1.1 header? Doesn't need to be there because it's already got a `pending` header :)